.copywrite.hcl

```diff
@@ -0,0 +1,31 @@
+schema_version = 1
+
+project {
+  license        = "MPL-2.0"
+  copyright_year = 2020
+
+  header_ignore = [
+    # changie tooling configuration and CHANGELOG entries (prose)
+    ".changes/unreleased/*.yaml",
+    ".changie.yaml",
+
+    # GitHub issue template configuration
+    ".github/ISSUE_TEMPLATE/*.yml",
+
+    # GitHub Actions workflow-specific configurations
+    ".github/labeler-*.yml",
+
+    # golangci-lint tooling configuration
+    ".golangci.yml",
+
+    # GoReleaser tooling configuration
+    ".goreleaser.yml",
+
+    # Release Engineering tooling configuration
+    ".release/*.hcl",
+
+    # Protoc generated files
+    "tfprotov5/internal/tfplugin5/*.pb.go",
+    "tfprotov6/internal/tfplugin6/*.pb.go",
```

Comment:
Hopefully `copywrite` should be happier now with the headers being included after auto-generation, but if not, let's followup after this PR. 👍 

```suggestion
```

Reply:
Removed ⚡